### DISCUSSION TITLE
toggle

### DISCUSSION THREAD

**Discussion by Plakman** [tajemniktv/TajsGraph#23] [Ideas]
Hey, 
would it be possible to add a toggle for the mod to be turned off and on?
i like how it looks but when i want more fps it would be nice if i could just disable lumen.
i'm not sure if that would break the mod.

so a button to easily reapply the mod settings after turning of lumen?

if it doesn't you can delete this :)
i'm getting about 60 fps on a 3080 without many buildings around.

thanks

**Comment by tajemniktv** [comment]
Hm, a complete "toggle on/off" might not be possible due to some vars require changing them before runtine, but I could think of some that are the biggest FPS killers. I do have plans for a dedicated UI with some settings (and maybe presets) https://github.com/tajemniktv/TajsGraph/issues/8
